internal/datastore/memdb/schema.go

```diff
@@ -101,30 +100,18 @@ func (r relationship) String() string {
 		caveat = "[" + r.caveat.caveatName + "]"
 	}
 
-	return r.namespace + ":" + r.resourceID + "#" + r.relation + "@" + r.subjectNamespace + ":" + r.subjectObjectID + "#" + r.subjectRelation + caveat
+	expiration := ""
+	if r.expiration != nil {
+		expiration = "[expiration:" + r.expiration.Format(time.RFC3339Nano) + "]"
```

Comment:
didn't we define a constant for the format we adopted somewhere? Please reuse to avoid drift

Reply:
We do, but I don't want to expose it in the tuple package. This is for debugging anyway and doesn't need to match the string format entirely (note it doesn't include caveat context)